### ISSUE TITLE
typo in import statement

### ISSUE THREAD

**Issue by orthodoc** [VirgilSecurity/e3kit-js#2]
In `EThree.ts` the second import statement

```ts
import VirgilToolbox from './virgilToolbox';
```
should be

```ts
import VirgilToolbox from './VirgilToolbox';
```

**Comment by orthodoc** [comment]
Easy PR. But I am not able to run the tests due to this error:
```
Cannot find module '../virgil_crypto_node.node' from 'virgil-crypto-pythia.cjs.js'

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:221:17)
      at Object.<anonymous> (node_modules/virgil-crypto/dist/virgil-crypto-pythia.cjs.js:5:11)
```
Resolved this by running: `npm install virgil-crypto --save` to install a node add on

Now the test suite has failing tests.

**Comment by xlwknx** [comment]
Could you show error logs from test?

**Comment by orthodoc** [comment]
```
> @virgilsecurity/e3kit@0.1.0 test /home/bdb/Projects/Dx/e3kit-js
> jest

 FAIL  src/__tests__/EThree.test.ts (64.763s)
  VirgilE2ee
    ✓ full integration test (11616ms)
  local bootstrap (without password)
    ✓ has local key, has no card (2636ms)
    ✓ has local key, has card (2013ms)
    ✓ has no local key, has card (1126ms)
    ✓ STA-1 has no local key, has no card (2275ms)
  remote bootstrap (with password)
    ✓ has no local key, has no card (4268ms)
    ✓ has no local key, has card (2684ms)
    ✓ wrong password (2353ms)
  lookupKeys
    ✓ STE-1 lookupKeys success (1356ms)
    ✓ lookupKeys nonexistent identity (735ms)
    ✓ lookupKeys with error (387ms)
    ✓ STE-2 lookupKeys with empty array of identities (10ms)
  change password
    ✓ should change password (7687ms)
    ✓ should change password faster if already bootstraped (5379ms)
  backupPrivateKey
    ✓ success (3625ms)
    ✓ fail (1135ms)
  encrypt and decrypt
    ✕ STE-3  (909ms)
    ✓ STE-4 encrypt for empty public keys (822ms)
    ✓ STE-5 decrypt for empty public keys (390ms)
    ✓ STE-6 encrypt and decrypt without public keys (404ms)
    ✕ STE-7 decrypt message without sign (959ms)
    ✓ STE-8 no decrypt/encrypt before bootstrap (20ms)
    ✕ should return buffer (881ms)
  cleanup()
    ✓ should delete key on logout (2569ms)
    ✓ reset backup private key (4198ms)

  ● encrypt and decrypt › STE-3 

    TypeError: Cannot read property 'catch' of undefined

       98 |         const responses = await Promise.all(
       99 |             identities.map(i =>
    > 100 |                 this.toolbox.getPublicKey(i).catch(e => (e instanceof Error ? e : new Error(e))),
          |                                             ^
      101 |             ),
      102 |         );
      103 | 

      at src/EThree.ts:100:45
      at Array.map (native)
      at EThree.<anonymous> (src/EThree.ts:99:24)
      at step (src/EThree.ts:32:23)
      at Object.next (src/EThree.ts:13:53)
      at src/EThree.ts:7:71
      at Object.<anonymous>.__awaiter (src/EThree.ts:3:12)
      at EThree.Object.<anonymous>.EThree.lookupPublicKeys (src/EThree.ts:177:16)
      at Object.<anonymous> (src/__tests__/EThree.test.ts:418:43)
      at step (src/__tests__/EThree.test.ts:43:23)

  ● encrypt and decrypt › STE-7 decrypt message without sign

    TypeError: Cannot read property 'catch' of undefined

       98 |         const responses = await Promise.all(
       99 |             identities.map(i =>
    > 100 |                 this.toolbox.getPublicKey(i).catch(e => (e instanceof Error ? e : new Error(e))),
          |                                             ^
      101 |             ),
      102 |         );
      103 | 

      at src/EThree.ts:100:45
      at Array.map (native)
      at EThree.<anonymous> (src/EThree.ts:99:24)
      at step (src/EThree.ts:32:23)
      at Object.next (src/EThree.ts:13:53)
      at src/EThree.ts:7:71
      at Object.<anonymous>.__awaiter (src/EThree.ts:3:12)
      at EThree.Object.<anonymous>.EThree.lookupPublicKeys (src/EThree.ts:177:16)
      at Object.<anonymous> (src/__tests__/EThree.test.ts:470:45)
      at step (src/__tests__/EThree.test.ts:43:23)

  ● encrypt and decrypt › should return buffer

    TypeError: Cannot read property 'catch' of undefined

       98 |         const responses = await Promise.all(
       99 |             identities.map(i =>
    > 100 |                 this.toolbox.getPublicKey(i).catch(e => (e instanceof Error ? e : new Error(e))),
          |                                             ^
      101 |             ),
      102 |         );
      103 | 

      at src/EThree.ts:100:45
      at Array.map (native)
      at EThree.<anonymous> (src/EThree.ts:99:24)
      at step (src/EThree.ts:32:23)
      at Object.next (src/EThree.ts:13:53)
      at src/EThree.ts:7:71
      at Object.<anonymous>.__awaiter (src/EThree.ts:3:12)
      at EThree.Object.<anonymous>.EThree.lookupPublicKeys (src/EThree.ts:177:16)
      at Object.<anonymous> (src/__tests__/EThree.test.ts:510:38)
      at step (src/__tests__/EThree.test.ts:43:23)

Test Suites: 1 failed, 1 total
Tests:       3 failed, 22 passed, 25 total
Snapshots:   0 total
Time:        65.118s
Ran all test suites.
```
All relate to inability to get the publicKey

**Comment by xlwknx** [comment]
I wasn't able reproduce this behavior, can you specify next information
1. do you use yarn or npm ?
2. Your platform
2. `node -v` command output
3. `npm -v` command output
4. `git diff 9fe8d2aadba04135d2a2ba20b2012809725ca406` command output

**Comment by orthodoc** [comment]
Fixed by commit ff441679df81fe72c73d757f233ee04b2648cd33

Not relevant any more. Thanks @xlwknx
